packages/rancher-eks-operator/rancher-eks-operator-crd/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/eks-operator/releases/download/v1.2.0/rancher-eks-operator-crd-1.2.0.tgz
```

Comment:
how is this still using 1.2.0 when we updated it to v1.2.1 (which is current latest) in https://github.com/rancher/charts/pull/2667/files#diff-a3151abc5568a26cdb9542c717ab32ef2c95c65e366c83cad4a5d9d2d625b7eaR1

Reply:
Maybe problem with github runner